content/blog/2021/04/2021-04-12-jenkins-operator-sub-project.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Jenkins Operator becoming sub-project!"
+tags:
+- kubernetes
+- operator
+- contributing
+authors:
+- antoniaklja
+- sylwiabrant
+sig: docs
+opengraph:
+  image: /images/post-images/2021-04-jenkins-operator/tshirt-logo.jpg
```

Comment:
It will look pretty bad as opengraph in social media.
We have a slidedeck with all opengraphs we use, e.g. here is a version from the Jenkins Operator online meetup with @tomaszsek last year: https://docs.google.com/presentation/d/1Q1PgNnRTgzBpVRXPqQo3PudzCa2eoc6_1_NRjFRMLrU/edit#slide=id.g76e0d29886_0_0